stdpopsim/cli.py

```diff
@@ -371,9 +371,7 @@ def add_simulate_species_parser(parser, species):
                 "Available maps: "
                 f"{', '.join(choices)}. "))
 
-    if len(species.genome.chromosomes) == 1:
-        species_parser.set_defaults(chromosome=species.genome.chromosomes[0].id)
-    else:
+    if len(species.genome.chromosomes) > 0:
```

Comment:
I think this statement will always be true. Codecov shows a yellow box for this line, which I think indicates that only one condition is tested.